nevergrad/benchmark/test_experiments.py

```diff
@@ -36,14 +36,18 @@ def test_experiments_registry(name: str, maker: tp.Callable[[], tp.Iterator[expe
     if all(x in name for x in ["image", "quality"]) and platform.system() == "Windows":
         raise SkipTest("Image quality not guaranteed on Windows.")
 
+    # ANM does not work under Windows.
+    if "gym" in name and platform.system() == "Windows":
+        raise SkipTest("Some gym problems suffer on Windows.")
+
     # Basic test.
     with tools.set_env(NEVERGRAD_PYTEST=1):
         with datasets.mocked_data():  # mock mlda data that should be downloaded
             check_maker(maker)  # this is to extract the function for reuse if other external packages need it
 
     # Some tests are skipped on CircleCI (but they do work well locally, if memory enough).
     if os.environ.get("CIRCLECI", False):
-        if any(name == x for x in ["images_using_gan", "mlda", "realworld"]):
+        if any(name == x for x in ["images_using_gan", "mlda", "mldakmeans", "realworld"]):
```

Comment:
this change is not doing anything
```suggestion
        if any(name == x for x in ["images_using_gan", "mlda", "realworld"]):
```

Reply:
It skips mldakmeans where the previous version did not skip mldakmeans ?

Reply:
ah sorry I misread, there is something wrong with this condition, from the start it should have been 
`if any(x in name for x in ["images_using_gan", "mlda", "realworld"])` and in this case mldakmeans would have been skipped. Looping over `x` has just no interest otherwise (it's equivalent to `if name in [...]`), so I guess the change should just be to convert `name == x` to `x in name`

Reply:
I've updated the suggestion